### PR TITLE
Remove redundant info about redact_info_log

### PR DESCRIPTION
**Pull request by shichun-0415**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Remove redundant info about redact_info_log

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v5.4 (TiDB 5.4 versions)
- [x] v5.3 (TiDB 5.3 versions)
- [x] v5.2 (TiDB 5.2 versions)
- [x] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hehechen — APPROVED** [review]

File: tiflash/tiflash-configuration.md [hunk 1]
```diff
@@ -207,11 +207,6 @@ delta_index_cache_size = 0
     # cert_path = "/path/to/tiflash-server.pem"
     ## Path of the file that contains X509 key in PEM format.
     # key_path = "/path/to/tiflash-server-key.pem"
-
-    ## New in v5.0. This configuration item enables or disables log redaction. If the configuration value
-    ## is set to true, all user data in the log will be replaced by ?.
-    ## Note that you also need to set security.redact-info-log for tiflash-learner's logging in tiflash-learner.toml.
-    # redact_info_log = false
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:214]: Should keep these lines.

Comment (shichun-0415) [reply tiflash/tiflash-configuration.md:214]: > Should keep these lines.

They are the same as Lines 198-201. Are you sure that we should keep them? Seems they are redundant.

Comment (JaySon-Huang) [reply tiflash/tiflash-configuration.md:214]: OK

File: tiflash/tiflash-configuration.md [hunk 2]
```diff
@@ -237,10 +232,10 @@ delta_index_cache_size = 0
     ## If you set it to 0ms, the optimization is disabled.
     store-batch-retry-recv-timeout = "4ms"
 [security]
-    ## New in v5.0. This configuration item enables or disables log redaction.
-    ## If the configuration value is set to "true",
+    ## This configuration item enables or disables log redaction.
+    ## If the configuration value is set to true,
     ## all user data in the log will be replaced by ?. The default value is false.
-    redact-info-log = false
+    redact_info_log = false
```
Comment (JaySon-Huang) [inline tiflash/tiflash-configuration.md:238]:
```suggestion
    redact-info-log = false
```

**Review by JaySon-Huang — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**